Documentation/compatibility/aspnet-472-compat-doc.md

```diff
@@ -0,0 +1,29 @@
+## "dataAnnotations:dataTypeAttribute:disableRegEx" app setting is on by default in .NET Framework 4.7.2
+
+### Scope
+Minor 
+
+### Version Introduced
+4.6.2
```

Comment:
@HongGit, I'm wondering if we need to have two app compat issues -- one for the 4.7.1 introduction of the switch, and the other for the change of the default. The basic problem is that, if we classify it as a 4.7.1 change, it won't surface in a 4.7.1 to 4.7.2 search. But that's where the change is going to be most noticeable, given that the switch is now opt-out rather than the previous opt-in. What do you think, @mjrousos?

Reply:
Since it's an opt-in switch, wouldn't a person moving from < 4.6.1 to 4.6.1 see no difference? If that's the case, then I think we only need one document (which can list 4.7.2 as the `version introduced` since that's the first time a user would be affected by this app without trying to use it). Even though the feature was added in 4.6.1, my understanding is that nothing breaking was introduced then. The `Version Introduced` is meant to indicate the NetFX version in which someone's app could be broken.

Reply:
OK. That sounds good. We'll leave it as 4.6.2, @HongGit and @mjrousos. I'll merge this PR.

Reply:
Shouldn't it be 4.*7*.2?

Reply:
Yes, it should be 4.7.2, @mjrousos. I have no idea of how I missed that. I'll fix it.

Reply:
I am not sure I follow your discussion.  The switch was introduced in .NET 4.6.1 release, the default behavior is changing in .NET 4.7.2 release.  @rpetrusha @mjrousos can you please elaborate why 4.6.2 and 4.7.1 getting into the discussion?

Reply:
Okay, I see @mjrousos's points on using .NET 4.7.2 as version introduced.  I am okay will that.